hf_vision_model_tfserving_gke/provision_gke_cluster.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+export GKE_CLUSTER_NAME=tfs-cluster
+export GKE_CLUSTER_ZONE=us-central1-a
+export NUM_NODES=2
+
+export NUM_CORES=8
+export MEM_CAPACITY=16384
+
+eval "$(cat ~/.bashrc)"
+export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+
+gcloud container clusters create $GKE_CLUSTER_NAME \
+    --zone=$GKE_CLUSTER_ZONE \
+    --machine-type=custom-$NUM_CORES-$MEM_CAPACITY \
+    --num-nodes=$NUM_NODES 
```

Comment:
Don't we need to specify the base machine type like E2 or something?

Reply:
We can do that, but this one says it more specifically 8vCPUand 16G Ram

Reply:
I am not talking about standard machine types. Here GKE is using E2 series machines by default. I am simply asking if there's a way to control that.